### DISCUSSION TITLE
有关自定义Captcha的问题

### DISCUSSION THREAD

**Discussion by chekast** [looly/hutool#2995] [Q&A]
在进行验证码生成的时候由于字间距默认按照总像素百分比进行划分的，导致最后一个验证码常常由于字符扭曲而无法完全显示。
文档中没有找到有关CodeGenerator的更多信息，怎么进行字间距自定义呢？

**Comment by looly** [comment]
这个貌似没有自定义间距。没有完全显示也算是防止ocr的方法吧?
